### ISSUE TITLE
Code previews in dark mode are light

### ISSUE THREAD

**Issue by gazayas** [bullet-train-co/showcase#48]
<img width="538" alt="dark mode showcase" src="https://user-images.githubusercontent.com/10546292/226077080-bdc74b3b-1bb2-44f6-b847-197b07819bb3.png">

I saw that we use the `sc-highlight` class in the following `app/views/showcase/engine/_sample.html.erb`:

```erb
<section class="sc-highlight sc-px-4 sc-py-2 sc-relative sc-overflow-y-auto hover:sc-select-all">
  <pre><%= sample.source %></pre>
</section>
```

I would expect adding the `dark` variant with some Tailwind to change the styling though, but it doesn't. Maybe there's something I'm missing concerning how highlighting in this repository works?

**Comment by kaspth** [comment]
Originally I'd left it like this because it didn't look like there was a Dark Mode shipped in Rouge's GitHub theme, which is what we use. But there was! I've fixed it in #50.

**Comment by kaspth** [comment]
@gazayas thanks for submitting this too! It's nice with the extra pair of eyes, so I ended up giving this another look 😄
